readme.md

```diff
@@ -1,60 +1,148 @@
 # Official Chia Docker Container
 
-## Basic Startup
-```
-docker run --name <container-name> -d ghcr.io/chia-network/chia:latest
-(optional -v /path/to/plots:/plots)
+## Quick Start
+
+These examples shows valid setups using Chia for both docker run and docker-compose. Note that you should read some documentation at some point, but this is a good place to start.
+
+### Docker run
+
+```bash
+docker run --name chia -d ghcr.io/chia-network/chia:latest --expose=8447 -v /path/to/plots:/plots
```

Comment:
```suggestion
docker run --name chia -d ghcr.io/chia-network/chia:latest --expose=8444 -v /path/to/plots:/plots
```

I think it probably makes sense for the documentation to default to 8444 as the port (since this is the port used for peers) - any other necessary ports can be opened using the same example by those that need them